zkevm-circuits/src/evm_circuit/execution/begin_tx.rs

```diff
@@ -357,7 +442,7 @@ impl<F: Field> ExecutionGadget<F> for BeginTxGadget<F> {
 
         // 3. Call to account with empty code.
         cb.condition(
-            and::expr([not::expr(tx_is_create.expr()), no_callee_code.clone()]),
+            and::expr([no_callee_code.clone(), tx_is_invalid.expr()]),
```

Comment:
I don't quite understand this change. The original one is `not create & no_callee_code` but it becomes `invalid tx & no_callee_code`. Why doest it become `invalid tx` and if it's for invalid tx case, is the tx still persistent [here](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1503/files#diff-8f5984af7a71d8fdfb2862e9655564673740a1d72df468ede0f01dbbd3ffba98R446-R451)?

Reply:
If it's invalid tx case, the tx won't be persistent and will be skipped if enable_skipping_invalid_tx is set to True.

Reply:
But the logic here is if it's a invalid tx, then do the following checks which include `require(is_persistent, 1)` as L449.